backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/TextBoundsService.java

```diff
@@ -90,4 +95,15 @@ public TextBounds getBounds(Label label) {
         return this.textBoundsProvider.computeBounds(label.getStyle(), label.getText());
     }
 
+    public TextBounds getTextMinimumBounds() {
+        // @formatter:off
+        LabelStyle labelStyle = LabelStyle.newLabelStyle()
+                .fontSize(16)
+                .color("#000000") //$NON-NLS-1$
+                .iconURL("") //$NON-NLS-1$
+                .build();
+        // @formatter:on
+        return this.textBoundsProvider.computeBounds(labelStyle, MINIMUM_TEXT_LABEL_CHARACTERS);
+    }
```

Comment:
You can use a constant for that instead of computing a hardcoded value like that.

Reply:
No need for a constant anymore